README.md

```diff
@@ -1,14 +1,15 @@
 `docopt.js` – command line option parser, that will make you smile
 ===============================================================================
 [![Build Status](https://travis-ci.org/Eyal-Shalev/docopt.js.svg?branch=master)](https://travis-ci.org/Eyal-Shalev/docopt.js)
-[![docopt.js v1.0.0](https://img.shields.io/badge/docopt.js-1.0.0-blueviolet)](https://github.com/Eyal-Shalev/docopt.js)
+[![docopt.js v1.0.0](https://img.shields.io/github/package-json/v/Eyal-Shalev/docopt.js?label=Docopt.js)](https://github.com)
 [![node >= 12.16.0](https://img.shields.io/badge/node-%3E%3D%2012.16.0-brightgreen)](https://nodejs.org/)
 [![deno >= 0.33.0](https://img.shields.io/badge/deno-%3E%3D0.33.0-black)](https://deno.land/)
+<!-- [![on npm](https://img.shields.io/npm/dw/docopt.js)](https://npmjs.com/package/docopt.js) -->
```

Comment:
FIY It will be a problem (during `require`) if the npm package will contain the file extension. 
```suggestion
<!-- [![on npm](https://img.shields.io/npm/dw/docopt)](https://npmjs.com/package/docopt) -->
```